any_parser/any_parser.py

```diff
@@ -187,6 +189,74 @@ def extract_key_value(
         else:
             return f"Error: {response.status_code} {response.text}", None
 
+    def resume_extract(
```

Comment:
nit: are we planning to add async as well?

Reply:
- Added async resume extract
- Renamed to `extract_resume_key_value`
- added unittest for async resume extract
- Tested sync and async resume extract on preprod-api